launcher/package/ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml

```diff
@@ -6,7 +6,7 @@
     <summary>Manage Minecraft instances with ease</summary>
     <description>
         <p>Overview</p>
-        <p>MultiMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
+        <p>UltimMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
```

Comment:
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by